djautotask/sync.py

```diff
@@ -73,6 +73,7 @@ def __init__(self):
 
 class Synchronizer:
     lookup_key = 'id'
+    db_lookup_key = lookup_key
```

Comment:
I don't see why we can't just use `lookup_key` here?

Reply:
Discussed in person. Thanks for clearing that up.

Reply:
Explained in person, but for posterity, lookup_key is already in use by Picklist synchronizers for api instance lookups, and the lookup for its api instance form and record form is different.